conf/application.conf

```diff
@@ -86,10 +96,8 @@ microservice {
       company-auth-frontend {
         external-url = "http://localhost:9025"
         sign-in.path = "/gg/sign-in"
-        sign-in.continue-url="http://localhost:9438/agent-mapping/start"
```

Comment:
A few things now gone from the config. We can clean this up from the different environments once they've been deployed. Another tech debt ticket?: 
"logoutRedirectUrl" (not sure where this was used, but it was removed from the FrontendModule)
"company-auth-frontend.sign-out.path"
"company-auth-frontend.sign-out.continue-url"

Reply:
yup things to change in the config, after this has been merged?

Reply:
will make a ticket for this

Reply:
We may not need a new ticket. We need to add the mongo db config to the app-configs anyway. We can remove these configs at that time.